src/main/java/com/iwaconsolti/league/controller/MatchController.java

```diff
@@ -0,0 +1,34 @@
+package com.iwaconsolti.league.controller;
+
+import com.iwaconsolti.league.model.MatchModel;
+import com.iwaconsolti.league.service.MatchService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/Match")
+@Slf4j
+public class MatchController {
+
+    private final MatchService service;
+    @Autowired
+    public MatchController(MatchService service) {
+        this.service = service;
+    }
+
+    @GetMapping("/team/{id}")
+    public MatchModel getMatchById(@PathVariable int id) {
```

Comment:
A TEam can have multiple matches.

This endpoint is returning the match by id, not the matches of a specific team